flake-modules/updates/efmls-configs.nix

```diff
@@ -0,0 +1,74 @@
+{
+  lib,
+  vimPlugins,
+  writeText,
+  pkgs,
+}:
+let
+  tools = lib.trivial.importJSON "${vimPlugins.efmls-configs-nvim.src}/doc/supported-list.json";
+  languages = lib.attrNames tools;
+
+  languageTools =
```

Comment:
`toLanguageTools` would be clearer imo

Or `toLangTools'`, showing it is a more abstract version of the curried function below